src/main/java/mate/academy/bookshop/model/Book.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.bookshop.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
```

Comment:
```suggestion
    @Column(nullable = false)
    private String title;
```
The same for other fields